applications/settings/desktop_settings/application.fam

```diff
@@ -7,6 +7,6 @@ App(
         "desktop",
         "gui",
     ],
-    stack_size=1 * 1024,
+    stack_size=2 * 1024,
```

Comment:
x2 increase?

Reply:
Was needed at some point, but can be reverted back safely now.